sites/svelte.dev/src/routes/docs/+layout.server.js

```diff
@@ -1,9 +1,17 @@
-import { get_docs_data, get_docs_list } from '$lib/server/docs/index.js';
-
 export const prerender = true;
 
-export function load({ url }) {
+export async function load({ url }) {
+	const is_docs_index = url.pathname === '/docs';
+
+	if (!is_docs_index) {
+		const { get_docs_data, get_docs_list } = await import('$lib/server/docs/index.js');
+
+		return {
+			sections: url.pathname === '/docs' ? [] : get_docs_list(get_docs_data())
```

Comment:
The condition will always be false since we wouldn't get in here if the pathname was `/docs`

Reply:
Oops! Good catch